css/src/components/dismiss.scss

```diff
@@ -0,0 +1,23 @@
+.dismiss {
+	@include transparent-effects;
+
+	display: inline-block;
+	width: 1em;
+	height: 1em;
```

Comment:
Should these also be `rem` 🤔

Reply:
No, these need to adjust per possible atomic usage.